Networking/Networking/ProductImageInBackground/ProductImageStatus.swift

```diff
@@ -86,6 +86,62 @@ public enum ProductImageStatus: Equatable, Codable {
         }
     }
 
+    public var isUploading: Bool {
+        if case .uploading = self {
+            return true
+        }
+        return false
+    }
+
+    public var isUploadFailure: Bool {
+        if case .uploadFailure = self {
+            return true
+        }
+        return false
+    }
+
+    public var siteID: Int64 {
+        switch self {
+        case .uploading(_, let siteID, _),
+            .remote(_, let siteID, _),
+            .uploadFailure(_, _, let siteID, _):
+            return siteID
+        }
+    }
+
+    public var productOrVariationID: ProductOrVariationID {
+        switch self {
+        case .uploading(_, _, let productID),
+             .uploadFailure(_, _, _, let productID),
+             .remote(_, _, let productID):
+            return productID
+        }
+    }
+
+    public var assetType: ProductImageAssetType? {
```

Comment:
Nit: maybe just `asset` should be enough?

Reply:
Yes, I named it `assetType` as a reminder of its model, `ProductImageAssetType`. However, `asset` is also clear enough. 👍